Classes/GlobalStateExplorers/FLEXInstancesTableViewController.m

```diff
@@ -74,21 +80,93 @@ + (instancetype)instancesTableViewControllerForInstancesReferencingObject:(id)ob
     FLEXInstancesTableViewController *instancesViewController = [[self alloc] init];
```

Comment:
We can change the init to take arguments so that you can move row title construction inside there. I don't think we have a reason for these to be publicly settable.